IllegalStateException no access to folder 'C:\Users\Johannes\IdeaProjects\apexcharts

Vaadin 14.0.6
ApexCharts 2.0.0.beta4
Compilation works fine, when I try to deploy on Tomcat I get the following error: 

```
Caused by: java.lang.IllegalStateException: Running project in development mode with no access to folder 'C:\Users\Johannes\IdeaProjects\apexcharts'.
Build project in production mode instead, see https://vaadin.com/docs/v14/flow/production/tutorial-production-mode-basic.html
	at com.vaadin.flow.server.DeploymentConfigurationFactory.verifyFolderExists(DeploymentConfigurationFactory.java:283)
	at com.vaadin.flow.server.DeploymentConfigurationFactory.readBuildInfo(DeploymentConfigurationFactory.java:222)
	at com.vaadin.flow.server.DeploymentConfigurationFactory.createInitParameters(DeploymentConfigurationFactory.java:166)
	at com.vaadin.flow.server.DeploymentConfigurationFactory.createPropertyDeploymentConfiguration(DeploymentConfigurationFactory.java:126)
	at com.vaadin.flow.server.startup.ServletDeployer$StubServletConfig.createDeploymentConfiguration(ServletDeployer.java:143)
	at com.vaadin.flow.server.startup.DevModeInitializer.onStartup(DevModeInitializer.java:216)
	at org.apache.catalina.core.StandardContext.startInternal(StandardContext.java:5132)
	at org.apache.catalina.util.LifecycleBase.start(LifecycleBase.java:183)
	... 27 more
```

my guess is that somewhere this folder is hardcoded...! :) 

Creating the directory C:\Users\Johannes\IdeaProjects\apexcharts  results in no errors!! :)

--- a/pom.xml
+++ b/pom.xml
@@ -132,6 +132,10 @@
                             <Vaadin-Package-Version>1</Vaadin-Package-Version>
                         </manifestEntries>
                     </archive>
+                    <!-- Generated file that shouldn't be included in add-ons -->
+                    <excludes>
+                        <exclude>META-INF/VAADIN/config/flow-build-info.json</exclude>
+                    </excludes>
                 </configuration>
             </plugin>
             <plugin>

